documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_idShortPath.adoc

```diff
@@ -12,9 +12,23 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur f
 
 === Format "Path" (IdShortPath Serialization) in JSON
 
-To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms of an idShortPath notation to be returned in an unnamed JSON-array. The notation differs depending on whether a SubmodelElementCollection or a SubmodelElementList is used. In the first case, the submodel element's idShort is separated by "." (dot) from top level down to child level. In the second case, square brackets with an index "[\<<index>>]" are appended after the idShort of the containing SubmodelElementList.
+To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms of an idShortPath notation to be returned in an unnamed JSON-array. The notation differs depending on whether a SubmodelElementCollection or a SubmodelElementList is present. In the first case, the submodel element's idShort is separated by "." (dot) from top level down to child level. In the second case, square brackets with an index "[<Index>]" are appended after the idShort of the containing SubmodelElementList.
 
-In the following example, where a request for idShort paths starts at _MySubmodelElementCollection_ with SerializationModifier level = deep, the list of idShort paths is returned as follows:
+[.underline]#Grammar:#
+
+[listing]
+....
+<idShortPath> ::= <idShort> {["." <idShort> | "["<Index>"]" ]}* 
```

Comment:
This grammar allows indices with leading zeros, e.g. 00000 or 0000012. Not sure if this is intentional and/or acceptable. An alternative grammar that does not allow these leading zeros could be
```<idShortPath> ::= <idShort> {["." <idShort> | "["<Index>"]" ]}*

<Index> ::= <Zero> | <PositiveNumber>

<PositiveNumber>  ::= <NonZeroDigit>{<Digit>}*
<Digit> ::= <Zero> | <NonZeroDigit>
<Zero>  ::= "0"
<NonZeroDigit>  ::= "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
```

Reply:
will change